applications/crossbar/src/modules/cb_port_requests.erl

```diff
@@ -598,7 +598,7 @@ load_port_request(Context, Id) ->
         'false' -> Context2;
         'true' ->
             Doc = cb_context:doc(Context2),
-            IsPrivateKeys = [<<"is_private">>, <<"superduper_comment">>],
+            IsPrivateKeys = [<<"superduper_comment">>, <<"is_private">>],
```

Comment:
This should be put into `kzd_comment:is_private` instead and `kzd_comment` used in this module.

Reply:
@icehess I like that suggestion :) any chance you could do that tomorrow?